cmd/frcli/node_audit.go

```diff
@@ -41,6 +71,12 @@ var onChainReportCommand = cli.Command{
 			Name:  "enable_fiat",
 			Usage: "Create a report with fiat conversions.",
 		},
+		cli.StringSliceFlag{
+			Name: "categories",
+			Usage: "A set of custom categories to create the " +
+				"report with, interpreted as category1:" +
+				"regex,regex",
+		},
```

Comment:
Would be very nice if there was a flag like "use-default-categories" that then uses a pre-defined set of patterns that contains what the labels look like in loop and other projects. That's what most people would want to use anyway, right?

Reply:
Great idea! Going to do it on a per-project basis, so I have added one for loop now. Can't wait to expand the list 😜